src/activities/drawletters/drawletters_dataset.js

```diff
@@ -0,0 +1,100 @@
+
+/* GCompris - drawletters_dataset.js
+ * Copyright (C) 2016 Nitish Chauhan <nitish.nc18@gmail.com>
+
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+function get() {
+    return [
+            {
+                "imageName1": "D2.svg",
+                "imageName2": "D1.svg",
+                "coordinates": [[134,43],[178,43],[217,42],[271,43],[322,43],[372,51],[411,67],[451,100],[468,135],[478,169],[484,214],[477,262],[458,302],[426,334],[379,351],[328,366],[262,369],[211,369],[169,363],[134,362],[134,310],[138,258],[138,220],[137,172],[138,119],[135,80],[134,43]],
+                "sound": "d"
```

Comment:
I would put the complete sound there, not just the letter. Else, it limits the sounds you can actually play (if for another activity you want to play a sound in a different folder, you can't)